kriswallsmith/buzz/0.17/config/packages/buzz.yaml

```diff
@@ -0,0 +1,30 @@
+# Read more about client options at: https://github.com/kriswallsmith/Buzz/blob/master/doc/client.md
+services:
+    _defaults:
+        public: false
+
+    # Clients
+    Buzz\Client\Curl:
+        arguments:
+            - timeout: 10
+
+    Buzz\Client\FileGetContents:
+        arguments:
+            - timeout: 10
+
+    Buzz\Client\MultiCurl:
+        arguments:
+            - timeout: 10
+
+    # Browser
+    Buzz\Browser:
+        arguments: ['@Buzz\Client\Curl']
```

Comment:
I would add a comment before this line to explain how to switch client (or before the client interface alias in case you create one to select the client)

Reply:
That is an excellent idea. Thanks!